.github/workflows/source/makeMakefileForClangTidy.py

```diff
@@ -14,40 +14,49 @@
 
 def makeMakefileForClangTidy(argv):
     parser = argparse.ArgumentParser()
-    parser.add_argument("--input",
-                        help="Ccache log file",
-                        default="ccache.log.txt")
-    parser.add_argument("--identifier",
-                        help="Unique identifier for finding compilation line in the log file",
-                        default="WarpX/Source")
+    parser.add_argument("--input", help="Ccache log file", default="ccache.log.txt")
+    parser.add_argument(
+        "--identifier",
+        help="Unique identifier for finding compilation line in the log file",
+        default="WarpX/Source",
+    )
     # We assume WarpX/Source can be used as an identifier to distinguish
     # WarpX code from amrex, openMPD, and cmake's temporary files like
     # build/CMakeFiles/CMakeScratch/TryCompile-hw3x4m/test_mpi.cpp
-    parser.add_argument("--output",
-                        help="Make file for clang-tidy",
-                        default="clang-tidy-ccache-misses.mak")
+    parser.add_argument(
+        "--output",
+        help="Make file for clang-tidy",
+        default="clang-tidy-ccache-misses.mak",
+    )
     args = parser.parse_args()
 
     fin = open(args.input, "r")
     fout = open(args.output, "w")
 
     fout.write("CLANG_TIDY ?= clang-tidy\n")
-    fout.write("override CLANG_TIDY_ARGS += --extra-arg=-Wno-unknown-warning-option --extra-arg-before=--driver-mode=g++\n")
+    fout.write(
+        "override CLANG_TIDY_ARGS += --extra-arg=-Wno-unknown-warning-option --extra-arg-before=--driver-mode=g++\n"
+    )
     fout.write("\n")
 
     fout.write(".SECONDEXPANSION:\n")
     fout.write("clang-tidy: $$(all_targets)\n")
     fout.write("\t@echo SUCCESS\n\n")
 
-    exe_re = re.compile(r" Executing .*? (-.*{}.*) -c .* -o .* (\S*)".format(args.identifier))
+    exe_re = re.compile(
```

Comment:
My own preference is allowing longer lines. Keeping this as one line is more succinct and easier to read (for me at least).

Reply:
We usually do not mingle with the default, which are currently around 100 characters (soft limitish with some heuristics). The bot auto-breaks them. This makes `diffs` on github easier to read.

We can increase that limit manually, but since a bot formats them we did keep the defaults on all other BLAST projects so far.